### ISSUE TITLE
JetAppointment: Multiple Separate Calendars requested feature

### ISSUE THREAD

**Issue by kalqlate** [CrocoBlock/suggestions#5091]
Firstly, SUPER GREAT JOB on ALL of your products AND support. Simply amazing! Kudos to ALL! 

Regarding JetAppointment Plugin and the need for Multiple Separate Calendars: 

I see that JetAppointment supports multiple providers, but chatting with your online support agent, I understand that it doesn't currently support multiple separate calendars. 

My application is to have users reserve time for different entities: facilities, different tools, and different manufacturing equipment. Each entity would have its own calendar, availability, and appointments. I proposed to your online chat support agent about using filtering if each entity is viewed as a provider, and that my application has the potential to grow to hundreds to thousands of entities. After a few moments, the chat agent informed me that that currently wasn't possible and that I should submit the idea as a new feature request.

So my request to the Crocoblock geniuses: add the functionality to create and assign separate calendars to separate entities.

Bonus if you can go a step further... the entities I mentioned above -- facilities, tools, manufacturing equipment -- are all actually BuddyPress Groups. I implement them as groups because of the need to allow ad hoc user/member association and robust conversation around each entity. If in your consideration of adding multiple separate calendars to your already fantastic JetAppointments, it would blow the minds of many if you made selection and assignment of BuddyPress groups (filterable by group-type) an additional source of service providers. 

------  UPDATE 03/30/2022 -  6:48 PM (CST)  ------

Ahhh, through a quick experiment, I see you have abstracted both provider and services to be any registered WP entity. Fantastic! -- you never know how someone is going to dream to use the appointments feature. However, TAKE NOTE, I discovered that there may be a clash between BuddyPress/BuddyBoss "groups" and LearnDash "groups". 

- When I disable all LearnDash plugins, the "groups" selection option disappears, but "group types" remains as BuddyPress/BuddyBoss is still enabled. (BuddyBoss in my case.) 
- It is clear that "group types" comes from BP/BB because "group types" selection option disappears when BP/BB plugin is disabled. 
- BuddyBoss in particular has options to sync BB and LD groups; therefore, the selectable "groups" option shown in JetAppointment coming only from LD might be due to BB deferring to LD to be the master for "groups". Not sure. 

If you plan to work on extending JetAppointment to providers each having their own separate calendar, please also look into how LearnDash and BuddyPress/BuddyBoss "groups" are handled when building your various selection lists.

-------------------------------------------------------

Thank you for your time and attention, and please... carry on with the creation of beautiful, amazing, and very necessary innovative products for the web design community.

David Williams

**Comment by Topicimes** [comment]
:+1: 

This ties in a bit with my idea that is in high demand: https://github.com/Crocoblock/suggestions/issues/3460

**Comment by kalqlate** [comment]
Yes, I see that. Thanks for letting me know. Separate calendars would be great for many reasons.
